### ISSUE TITLE
Collect Dataset

### ISSUE THREAD

**Issue by subhasisj** [subhasisj/face-mask-detection#1]
Collect dataset from various resources and make sure images consist of people faces of majorly Indian Ethnicity

**Comment by subhasisj** [comment]
Collected data from https://www.pyimagesearch.com/2020/05/04/covid-19-face-mask-detector-with-opencv-keras-tensorflow-and-deep-learning/

**Comment by subhasisj** [comment]
Number Masked Faces samples: 686
Number Non-Masked Faces samples: 686
